docs/typography/examples/types-page.tsx

```diff
@@ -78,7 +84,7 @@ const TypesPage: React.FC = () => {
           <TypeCard {...TYPE_CARD_REGULAR_MERRIWEATHER_PROPS} />
         </Grid>
         <Grid item xs={12}>
-          <Typography sx={{ color: '#FFFFFF' }}>Examples</Typography>
+          <Typography sx={{ color: `${palette.common.white}` }}>Examples</Typography>

```

Comment:
Question - what is Typography defaulting its color to?

Reply:
It depends on the dark or light mode of theme, and in the documentation, it is generaly the white color.